core/vcpu.c

```diff
@@ -3763,7 +3779,15 @@ static int handle_msr_write(struct vcpu_t *vcpu, uint32_t msr, uint64_t val,
             break;
         }
         case IA32_CR_PAT: {
+            //Attempting to write an undefined memory type encoding into the PAT causes a
+            //general-protection (#GP) exception to be generated.
```

Comment:
Add a space after slashes and pay attention to the line wrap to ensure the long line does not exceed column 80.

`// Attempting to write an undefined memory type encoding into the`
`// PAT causes a general-protection (#GP) exception to be generated.`